src/test/java/io/candydoc/plugin/save_documentation/html/SaveDocumentationAsHTMLTest.java

```diff
@@ -82,7 +86,8 @@ class SaveDocumentationAsHTMLTest {
   private final List<Event> emptyBoundedContext =
       List.of(
           BoundedContextFound.builder()
-              .name("io.emptyBoundedContext")
+              .simpleName("emptyBoundedContext")
+              .canonicalName("io.emptyBoundedContext.package-info")
```

Comment:
io.empty_bounded_context => ensure the package you test has underscore to follow java convention